community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/prettifier/Prettifier.scala

```diff
@@ -148,7 +149,8 @@ class PrettifierParser extends Parser with Base with Strings {
       keyword("ASSERT") |
       keyword("SCAN") |
       keyword("CALL") |
-      keyword("UNION")
+      keyword("UNION") |
```

Comment:
That's what I get for having written a test case that tests two keywords in one go... Thanks for catching it :)